influxql/iterator.gen.go.tmpl

```diff
@@ -657,6 +691,7 @@ func (itr *{{$k.name}}InterruptIterator) Next() *{{$k.Name}}Point {
 type {{$k.name}}AuxIterator struct {
 	input      *buf{{$k.Name}}Iterator
 	output     chan *{{$k.Name}}Point
+	err        chan error
 	fields     auxIteratorFields
 	background bool
 }
```

Comment:
Done. I also found a bug where I forgot to propagate the error to the underlying channel iterators.